### DISCUSSION TITLE
DB Seed - Typescript - MongoDB - Unexpected token

### DISCUSSION THREAD

**Discussion by NicolasMarafetti** [prisma/prisma#14779] [Q&A] [answered]
**Technology**
Next.js / Typescript

**Documentation**
https://www.prisma.io/docs/guides/database/seed-database.

**package.json**
```
{
  ...
  "prisma": {
    "seed": "ts-node --compiler-options {\"module\":\"CommonJS\"} prisma/seed.ts"
  },
  ...
}
```
```Shell
npx prisma db seed
```
```
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".ts" for D:\Programmation\Websites\nico-piece-auto\website\prisma\seed.ts
```

**seed.ts**
```javascript
import { PrismaClient } from '@prisma/client';
import { CRYPTO_SECRET_KEY } from '../config/index';

const CryptoJS = require('crypto-js');
const prisma = new PrismaClient();

async function main() {

  const alicePasswordEncrypted = CryptoJS.AES.encrypt(
    'Alice',
    CRYPTO_SECRET_KEY
  ).toString();

  await prisma.user.create({
    data: {
      admin: false,
      email: 'alice@prisma.io',
      firstName: 'Alice',
      password: alicePasswordEncrypted,
      validated: true
    },
  })

  const bobPasswordEncrypted = CryptoJS.AES.encrypt(
    'Bob',
    CRYPTO_SECRET_KEY
  ).toString();

  await prisma.user.create({
    data: {
      admin: false,
      email: 'bob@prisma.io',
      firstName: 'Bob',
      password: bobPasswordEncrypted,
      validated: true
    },
  })

}

main()
  .then(async () => {
    await prisma.$disconnect()
  })
  .catch(async (e) => {
    console.error(e)
    await prisma.$disconnect()
    process.exit(1)
  })
```

**Other test with package.json**
If I use ```"seed": "ts-node prisma/seed.ts"``` I have the same error

**Comment by NicolasMarafetti** [comment, marked as answer]
I had to remove 
```
"type": "module"
```
from the package.json file

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
